README.md

```diff
@@ -60,4 +68,49 @@ Get in touch today! [Book a demo](https://xygeni.io/book-a-demo?utm_source=jenki
     * Set the `Xygeni API Url` (https://api.xygeni.io).
     * Optionally click the `TestToken` button to check url connection and token validity.
     * Click the `Save` button. 
-    ![Xygeni Configuration](docs/images/xygeni-config.png)
\ No newline at end of file
+    ![Xygeni Configuration](docs/images/xygeni-config.png)
+
+
+
+# Xygeni Sensor Unusual Activity
+
+By installing the Xygeni sensor, user activities and actions will be monitored and suspicious events will be sent to the Xygeni server for analysis.
+
+
+# XygeniSalt Step
```

Comment:
I recommend givint the section a different title (`XygeniSalt` is the command name) 

# Step for building a SLSA provenance attestation

Reply:
The title was updated and the command is now `XygeniSaltSlsa`